### PR TITLE
Copy resource files to static directory

### PR DESCRIPTION
**Pull request by takuan-osho**
This PR is for https://github.com/sphinx-doc/sphinx/issues/4064.

I used https://github.com/shimizukawa/sphinx-websupport-app as a test of this PR.

If this PR is merged, Sphinx core could remove resource files such as `comment-bright.png` from Sphinx itself.

### CODE REVIEW

File: sphinxcontrib/websupport/builder.py [hunk 1]
```diff
@@ -178,6 +178,25 @@ def handle_finish(self):
                     shutil.rmtree(dst)
                 shutil.move(src, dst)
 
+        # copy resource files to static dir
```
Comment (tk0miya) [inline sphinxcontrib/websupport/builder.py]: Could you move this block to a new method please?

File: sphinxcontrib/websupport/builder.py [hunk 2]
```diff
@@ -178,6 +178,25 @@ def handle_finish(self):
...
+        resources = [
```
Comment (tk0miya) [inline sphinxcontrib/websupport/builder.py]: It would be nice to define this as a constant.

File: sphinxcontrib/websupport/builder.py [hunk 3]
```diff
@@ -178,6 +178,25 @@ def handle_finish(self):
...
+            'ajax-loader.gif',
+            'comment-bright.png',
+            'comment-close.png',
+            'comment.png',
+            'down-pressed.png',
+            'down.png',
+            'up-pressed.png',
+            'up.png',
+        ]
+        package_dir = path.abspath(path.dirname(__file__))
```
Comment (tk0miya) [inline sphinxcontrib/websupport/builder.py]: I think package_dir would be commonly used. How about moving this to module global?

File: sphinxcontrib/websupport/builder.py (hunk 2 above)
Comment (takuan-osho) [reply sphinxcontrib/websupport/builder.py]: I got it. I'll fix it.

File: sphinxcontrib/websupport/builder.py (hunk 1 above)
Comment (takuan-osho) [reply sphinxcontrib/websupport/builder.py]: I fixed it.

File: sphinxcontrib/websupport/builder.py (hunk 3 above)
Comment (takuan-osho) [reply sphinxcontrib/websupport/builder.py]: I fixed it.

File: sphinxcontrib/websupport/builder.py (hunk 2 above)
Comment (takuan-osho) [reply sphinxcontrib/websupport/builder.py]: I fixed it.

**Pull request merged**